magento/magento2-page-builder
# magento/magento2-page-builder#609: Content Type Toolbar hidden under …

…PageBuilder header in Full screen - Fixed Content Type Toolbar translateY calculation





### Description (*)


### Story

* magento/magento2-page-builder#609: Content Type Toolbar hidden under PageBuilder header in Full screen
* magento/magento2-page-builder#614: Deprecate OpenPageBuilderFullScreen Action Group
* magento/magento2-page-builder#613: Sliders do not render correctly after toggle a screen mode several times
* magento/magento2-page-builder#612: Multiple row with parallax background does not render correctly after toggle a screen mode several times

### Builds


### Related Pull Requests



### Manual testing scenarios (*)

1. Browse **Stores** >> **Configuration** >> **Content Management** and make sure option "Enable WYSIWYG Editor" is set to "Enabled by Default".
2. Navigate to Admin page with PageBuilder.
3. Open PageBuilder in Full Screen.
3. Add Text content type to Row and click on the added Content Type.
4. If there is not enough space on the top of the page, Text Content Type Option Panel should appear below the Content Type textarea. 
5. Add Column Content Type to Row and move previously added Text to one of the columns. Check again if Options Panel is not hidden under PageBuilder header.
6. Check **step 4** again, but with different Column sizes: 3/12, 5/12, 7/12, 9/12, 11/12.
7. Save page and check Options Panel again.
8. Remove Text Content Type and add Heading Content Type.
9. Repeat **steps 4 - 7** but for Heading Content Type.

### Questions or comments


### Checklist

---

### Test Plan
#### Manual Checks
- [x] Verify issue does not occur in top Row in full screen for
  - [x] Text tinyMCE editor
  - [x] Banner tinyMCE editor
  - [x] Slide tinyMCE editor
  - [x] Check each of above for smaller viewports such that the tinyMCE editor toolbar increases in height
- [x] Verify issue does not occur in Column for
  - [x] Text tinyMCE editor
  - [x] Banner tinyMCE editor
  - [x] Slide tinyMCE editor
  - [x] Check each of above for smaller viewports such that the tinyMCE editor toolbar increases in height
- [x] Verify issue does not occur in Row for Heading
  - [x] Check each of above for smaller viewports such that the tinyMCE editor toolbar increases in height
- [x] Verify issue does not occur in Column for Heading
  - [x] Check each of above for smaller viewports such that the tinyMCE editor toolbar increases in height
- [x] Verify each above scenario in

#### MFTF Tests
Make sure we have test coverage for
* Text in top row & make sure issue doesnt happen
* Heading in top row & make sure issue doesnt happen
* Text in smaller column & make sure issue doesnt happen
* Heading in smaller column & make sure issue doesnt happen

#### Hip Tests
None
